spec.md

```diff
@@ -451,7 +451,7 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id?acc
 ### Response
 
 | Status Code | Description |
-|---|---|
+| --- | --- |
```

Comment:
I have never seen spaces between the dashes and pipes in a markdown header before in my life.... citation?

#buildthebikeshed